gitlab/hatch.toml

```diff
@@ -13,6 +13,7 @@ matrix.version.env-vars = [
   { key = "GITLAB_VERSION", value = "13.12.15-ce.0", if = ["13.12"] },
   { key = "GITLAB_VERSION", value = "14.10.5-ce.0", if = ["14.10"] },
   { key = "GITLAB_VERSION", value = "15.10.0-ce.0", if = ["15.10"] },
+  { key = "GITLAB_IMAGE", value = "yrzr/gitlab-ce-arm64v8", platform_machine = "arm64"}
```

Comment:
where is the default value defined?

Reply:
I did not define it explicitly in the `hatch`, the default value is [here](https://github.com/DataDog/integrations-core/pull/14397/files#diff-1e942fa5b4c7f35a51ae69345245006e1820cb8a90b9b2226c48b851befc882dR4). I can add it to hatch for clarity

Reply:
Oh, I didn't know that's how defaults worked on docker files